### ISSUE TITLE
Error with openssl-dev and libressl-dev during install

### ISSUE THREAD

**Issue by hlharris22** [amitn322/blackesk#14]
First off thanks for this great repository. It's really neat

Docker Version: 20.10.21
Docker-compose version 1.29.2
NAME="Ubuntu"
VERSION="20.04.2 LTS (Focal Fossa)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 20.04.2 LTS"
VERSION_ID="20.04"

I am however running into some issues when attempting to use the install.sh. The issue occurs for both the single-node and multi-node.

![image](https://user-images.githubusercontent.com/58747147/206544765-502217e3-a887-48e4-8b53-af239f5c6ed0.png)

This error leads to the syslog-ng failing to build.

![image](https://user-images.githubusercontent.com/58747147/206546665-69682747-a439-4906-b0b2-a32bcd64c561.png)

Any help would be greatly appreciated!

**Comment by StoryDevOffical** [comment]
yes this error is triggered when trying to install **curl-dev** in **build.sh**

now, it seems there are some problem with libressl-dev in latest alpine release.
Like Dockerfile point to latest (no tag), try to use [alpine 3.11.11](https://hub.docker.com/layers/library/alpine/3.11.11/images/sha256-b2ae6c78091b75954894951fc0b0f6ba2f4997f98b1bb556fe648fb71b5cd370?context=explore).

It can have later version which work but i don't try.

**Comment by amitn322** [comment]
@StoryDevOffical thanks for pointing that out, I will test it out and update @hlharris22

**Comment by amitn322** [comment]
@hlharris22 @StoryDevOffical  This issue has been fixed. Can you please check and confirm ?

**Comment by hlharris22** [comment]
@amitn322 Thanks for fixing. I'll test it out shortly and get back to you

**Comment by VishwaJagdale** [comment]
@amitn322 Today, I tried on below configuration but not worked with latest alpine version. Error was **"Failed to solve: process "/bin/sh -c /build.sh" did not complete successfully: exit code: 1"**.   

After RnD for 4-5hrs gone through suggestion of @StoryDevOffical and tried with alpine version 3.11.11 and its working with **alpine version 3.11.11 only.** 

OS - Ubuntu Version 22.04.2 LTS (Jammy Jellyfish)
Kernel Version: 5.19.0-41-generic
Docker Engine - Community
Docker Version: 24.0.1, build 6802122
Docker-compose version 1.29.2

**Comment by ErickPH** [comment]
@amitn322 I Installed Alpine Standard 3.19.0 and then edited the syslog build file and removed the installation of libressl-dev and uncomment openssl-dev.
As of Dec 07 2023, there is no reason to use libressl-dev instead of openssl-dev.
So far, the build process ended up correctly.

**Comment by amitn322** [comment]
Thank you, I will update that in the code.
